src/d/actor/d_a_player_main_data.inc

```diff
@@ -73,7 +73,7 @@ static dCcD_SrcCyl l_cyl_src = {
     },
 };
 
-static dCcD_SrcCyl l_wind_cyl_src = {
+dCcD_SrcCyl l_wind_cyl_src = {
```

Comment:
what's the reasoning for removing this keyword? if you're looking at the green g in objdiff, that's just dtk's assumption for main.dol symbols because framework.map doesn't tell us if it had the static keyword or not. but in RELs they usually do have it.